qiskit/visualization/__init__.py

```diff
@@ -110,6 +110,8 @@
 
 from qiskit.visualization.transition_visualization import visualize_transition
 from .pulse_visualization import pulse_drawer
```

Comment:
The docs for Pulse Visualization can be added when the API docs are generated. For doing so, add the below code to the docstrings in this file:
```
Pulse Visualizations
====================

.. autosummary::
   :toctree: ../stubs/

   pulse_drawer

.. autosummary::
   :hidden:
   :toctree: ../stubs/

   qiskit.visualization.pulse.qcstyle
   qiskit.visualization.pulse.interpolation
```

Screenshot:
<img width="1355" alt="image" src="https://user-images.githubusercontent.com/11485594/75287748-f0129400-57e8-11ea-9194-c547189661bf.png">